recipe/meta.yaml

```diff
@@ -81,9 +85,12 @@ about:
     This package provides a set of python utilities for gravitational-wave
     detector characterisation and data quality. It is intended for use by
     the International Gravitational Wave Network (IGWN) of ground-based
-    detectors, along with the Omicron (C++) package.
+    detectors, along with the Omicron (C++) package. Documentation is
+    available at https://pyomicron.readthedocs.io
```

Comment:
You should be able to add `doc_url` as a key equivalent to `dev_url`, rather than plaintxt in the description.

Reply:
I did not look hard enough for the about element list. My first guess produced an error.